qm2/scf.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import psi4
+
+np.set_printoptions(suppress=True, precision=4)
+
+
+def build_geom(mol_string):
+    mol = psi4.geometry(mol_string)
+    
+    # Build a molecule
+    mol.update_geometry()
+    mol.print_out()
+    return mol 
+
+e_conv = 1.e-6
+d_conv = 1.e-6
+nel = 5
```

Comment:
the number of electrons will have to be set programmatically too, if we are going to allow the user to pass a different molecule it might have a different number of doubly occupied orbitals!